thrall/app/ThrallComponents.scala

```diff
@@ -53,9 +55,10 @@ class ThrallComponents(context: Context) extends GridComponents(context, new Thr
 
   val highPrioritySource: Source[KinesisRecord, Future[Done]] = KinesisSource(highPriorityKinesisConfig)
   val lowPrioritySource: Source[KinesisRecord, Future[Done]] = KinesisSource(lowPriorityKinesisConfig)
+  val reingestionSource: Source[(UpdateMessage, Instant), Future[Done]] = ReingestionSource(???)
```

Comment:
think this should become `migrationSource` based on https://github.com/guardian/grid/pull/3324

Reply:
decided to do this as part of https://github.com/guardian/grid/pull/3324 instead, as a follow-up